packages/react/src/experimental/Forms/EntitySelect/index.stories.tsx

```diff
@@ -702,3 +702,60 @@ export const WithActions = {
     )
   },
 }
+
+export const WithCreate = {
```

Comment:
can we make it open by default in the story? this way we can capture a screenshot with the item